walkthroughs/publishing-walkthroughs/walkthrough.adoc

```diff
@@ -62,28 +66,24 @@ You have the URL one or more repositories with the exact content you want to pub
 Continue searching, new walkthroughs are created on GitHub monthly and the link:{linkIndexURL}[index of walkthrough repositories] is updated. 
 
 [time=10]
-== Publishing the Walkthrough in the Webapp
-
-You can add repositories containing one or more Walkthroughs to a Webapp deployed to Openshift.
-
-=== Publishing a git repository to the Webapp
+== Adding a git repository to the Webapp
 
 For each repository URL that you have from Task 1:
 
 . Open the link:{openshift-host}[Openshift Console].
 
-. Find and open the *webapp* project.
-
-. Find and open the *Resources* menu on the left.
+. Navigate to the *webapp* project.
 
-. Select the *Other Resources* menu item.
+. Choose *Other Resources* from the *Resources* menu.
 
-. Select *Web App* in the resources drop down.
+. Choose *Web App* from the *Other Resources* menu. 
+The tutorial-web-app-operator entry is displayed.
 
-. Click *Actions* and select *Edit YAML* in the resulting drop-down menu.
+. Choose *Edit YAML* from the *Actions* menu.
+An editor is displayed.
 
-. Find the *Spec* > *Template* > *Parameters* entry in this YAML
-. Add (or edit, if it already exists) a parameter:
```

Comment:
So are we sure the env var will always be there? I think I've seen the case when it wasn't available.

Reply:
It won't at first but I thought it was intuitive enough to add it by saying "edit"

Reply:
It doesn't seem to me intuitive TBH. We are saying `Edit the `WALKTHROUGH_LOCATIONS` parameter` but the parameter doesn't exist right? But yeah it's likely just minor.

Reply:
+1 @tremes  I've changed it to `Create or edit`